lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -57,12 +57,14 @@
                          inProgressCount = this.inProgressCourseCollection.length,
                          remainingCount = this.remainingCourseCollection.length,
                          totalCount = completedCount + inProgressCount + remainingCount,
+                         buyButtonUrl = this.options.urls.buy_button_url + '&bundle='
+                                        + encodeURIComponent(this.options.programData.uuid),
```

Comment:
Yes there always should be for this view, but I could also add a check for that.